docs/Makefile

```diff
@@ -88,11 +92,11 @@ multiversionpreview: multiversion
 
 # Test commands
 .PHONY: test
-test: setup
+test: setup build-typedoc
 	$(SPHINXBUILD) -b dirhtml $(TESTSPHINXOPTS) $(BUILDDIR)/dirhtml
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/dirhtml."
 
 .PHONY: linkcheck
-linkcheck: setup
+linkcheck: setup build-typedoc
```

Comment:
We shouldn't rebuild the typedoc docs when we run `make linkcheck`.

Reply:
Actually, in general I'm wondering if we need to rebuild the typedoc docs whenever we build the rest of the site, or if we can instead include it as a separate `make` command that you only run when you're working specifically on those docs, or as a first step if there EDSL docs are not present in `aerie/schedule-server/scheduling-dsl-compiler/build/docs/scheduling-dsl-compiler` ([which appears to be possible to check with MAKEFILE syntax ](https://stackoverflow.com/questions/5553352/how-do-i-check-if-file-exists-in-makefile-so-i-can-delete-it)using the [`wildcard`](https://www.gnu.org/software/make/manual/html_node/Wildcard-Function.html) command).

Reply:
For ensuring that we publish an up-to-date version, we can modify the GitHub workflow to always build the EDSL docs before running `make multiversion`. My concern here was on the increased build time for devs working on the docs, especially since the EDSL files are autogenerated. It's not PR-blocking, but the difference felt significant enough to bring up.

Reply:
I think that's a fine solution.